### DISCUSSION TITLE
add AllowedIps into the wg0.conf

### DISCUSSION THREAD

**Discussion by volodin73** [wg-easy/wg-easy#1530] [Q&A]
Hi, please explain how to add specific networks for a client (e.g., 10.10.0.16). The client is a MikroTik router. How can this be done through editing the docker-compose.yml file?"
![wg](https://github.com/user-attachments/assets/fcb99d34-80f2-43f9-ab14-25b60ab56c09)

**Comment by haitham506** [comment]


**Reply by kaaax0815** [reply]
please don't ping unnecessarily. will be implemented in v15
